flask_saml_sso/__init__.py

```diff
@@ -0,0 +1 @@
+from .base import *
```

Comment:
I believe `import *` is a bad idea, even in `init` files. In the UBS project we've also found that it may give unexpected problems because imports in the source file are also reimported. 

It's better to import the elements you want to expose one by one.